### PR TITLE
docs(installation): improve guides

### PR DESCRIPTION
**Pull request by eliobischof**
Follow-up issues
- [ ] #3824
- [ ] #3825 
- [ ] #3826 
- [ ] #3828 
- [ ] #3816

### CODE REVIEW

File: docs/docs/guides/installation/custom-domain.md [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+
+<!-- TODO: verify -->
+ZITADEL creates random subdomains for each instance created.
+However, for the default instance, this is most probably not the desired behavior.
+Your can give the default instance an explicit fixed custom domain here.
+
+```yaml
+S3DefaultInstance:
+  CustomDomain: 'zitadel.my.domain'
+```
```
Comment (eliobischof) [inline docs/docs/guides/installation/custom-domain.md:34]: @livio-a did I explain this correctly and completely?

**Comment by fforootd** [comment]
BTW I think we should add a persistent volumes to the containerised CockroachDB

**Comment by fforootd** [comment]
> > BTW I think we should add a persistent volumes to the containerised CockroachDB 
> 
> Already done that 😉

Awesome 😎

**Comment by fforootd** [comment]
Is this PR blocked by something? Can I assist?

**Comment by adlerhurst** [comment]
> Is this PR blocked by something? Can I assist?

@hifabienne is reviewing it.

**Comment by fforootd** [comment]
> > Is this PR blocked by something? Can I assist?
> 
> @hifabienne is reviewing it.

ok thanks

File: docs/docs/guides/installation/get-started/macos.mdx [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+```bash
+export MY_ARCHITECTURE="arm64 or amd64 depeding on your mac"
+curl -s https://api.github.com/repos/zitadel/zitadel/releases/tags/v2.0.0-v2-alpha.24 | grep "browser_download_url.*zitadel_Darwin_${MY_ARCHITECTURE}.tar.gz" | cut -d '"' -f 4 | xargs wget -qO - | sudo tar --extract --gzip --overwrite --directory /usr/local/bin zitadel && sudo chown $(id -u):$(id -g) /usr/local/bin/zitadel
+```
+
+Run ZITADEL
+
+```bash
+ZITADEL_EXTERNALSECURE=false zitadel start-from-init --masterkey "MasterkeyNeedsToHave32Characters"
+```
```
Comment (hifabienne) [inline docs/docs/guides/installation/get-started/macos.mdx:33]:
```suggestion
```bash
ZITADEL_EXTERNALSECURE=false zitadel start-from-init --masterkey "MasterkeyNeedsToHave32Characters" --tlsMode disabled
```
```

I followed the instructions and got the following error:
FATA[0000] unable to start zitadel                       caller="/home/runner/work/zitadel/zitadel/cmd/start/start_from_init.go:41" error="TLS is enabled: please specify a key (path) and a cert (path) or disable TLS if needed (e.g. by setting flag `--tlsMode external` or `--tlsMode disabled"

File: docs/docs/guides/installation/get-started/linux.mdx [hunk 3]
```diff
@@ -0,0 +1,17 @@
...
+
+```bash
+# Run a CockroachDB instance
+cockroach start-single-node --insecure --background --http-addr :9090
+
+# Download the zitadel binary
+curl -s https://api.github.com/repos/zitadel/zitadel/releases/tags/v2.0.0-v2-alpha.24 | grep "browser_download_url.*zitadel_Linux_$(uname -i).tar.gz" | cut -d '"' -f 4 | xargs wget -qO - | sudo tar --extract --gzip --overwrite --directory /usr/local/bin zitadel && sudo chown $(id -u):$(id -g) /usr/local/bin/zitadel
+
+# Run the zitadel binary
+ZITADEL_EXTERNALSECURE=false zitadel start-from-init --masterkey "MasterkeyNeedsToHave32Characters"
```
Comment (hifabienne) [inline docs/docs/guides/installation/get-started/linux.mdx:16]: Do we need to add --tlsMode disabled here as well?

Comment (eliobischof) [reply docs/docs/guides/installation/get-started/linux.mdx]: yes, done

**Review by hifabienne — APPROVED** [review]

**Pull request merged**